pallets/detach/src/lib.rs

```diff
@@ -124,19 +133,22 @@ impl<T: SigningTypes> SignedPayload<T> for DetachInternalData<T::Public> {
 	}
 }
 
+/// ¿
```

Comment:
This is the original data that was used to validate the link, this data is produced off-chain, e.g. for ethereum it would be a validator account signing a message